test/aries-js-worker/test/verifiable/verifiable.js

```diff
@@ -46,7 +57,7 @@ describe("Verifiable Store", function () {
         aries.destroy()
     })
 
-    it("Alice stores the verifiable credential received from the college", function (done) {
+    it(modePrefix + "Alice stores the verifiable credential received from the college", function (done) {
```

Comment:
nitpick: add an extra whitespace between the mode prefix and the rest of the description

Reply:
It's there in line 46 - https://github.com/hyperledger/aries-framework-go/pull/1410/files#diff-307e2ee4ae6edb8abe0a65e7a61b69e6R46